src/Discord.Net.WebSocket/BaseSocketClient.Events.cs

```diff
@@ -367,5 +367,22 @@ public partial class BaseSocketClient
             remove { _recipientRemovedEvent.Remove(value); }
         }
         internal readonly AsyncEvent<Func<SocketGroupUser, Task>> _recipientRemovedEvent = new AsyncEvent<Func<SocketGroupUser, Task>>();
+
+        //Invites
+        /// <summary> Fired when an invite is created. </summary>
```

Comment:
@Still34 We've become inconsistent with how the doc comments are written. Should we start asking for one specific formatting, and if so, which one?

Reply:
Changed the summary and remarks so they are more descriptive and easier to understand what's being passed like the Channel and Message events.

Reply:
> @Still34 We've become inconsistent with how the doc comments are written. Should we start asking for one specific formatting, and if so, which one?

I might be late to the party but I'm not spotting the inconsistency?

Reply:
We have `<summary> Text <summary>` (one line) and
```
<summary>
    Text
<summary>
```
(multiple lines, indented) both used in this PR alone.

Reply:
Personally, I'm leaning towards the latter. Although it takes up more space, it is easier to read when writing documentation. Perhaps we could open a new issue and or PR that addresses this in our contribution guide.

Reply:
Changed all summaries to use the latter.